### PR TITLE
feat: add korean language support

### LINKED ISSUE #178: feature: support Korean language 'ko-KR'
Is your feature request related to a problem? Please describe.
Currently Schedule-X doesn't support Korean language.

Describe the solution you'd like
I want to add Korean translation into Schedule-X (packages/translations/src/locales/ko-KR)

Additional context
I have cloned great project and use another language locale for a example.

### PR DESCRIPTION
**Pull request by Dodo9128**
## Checklist

Please put "X" in the below checkboxes that apply:

- [ ] If committing a change of production code, I have tested it in different browsers (Chrome, Firefox, Safari). 
- [x] If committing a new feature, I have first submitted an issue (Please note: you are free to open PRs for non-issued features, but opening an issue increases your chance of a successful PR). 
- [ ] If committing a new feature, I have also written the appropriate tests for it. 
- [x] I have tried to build the feature in a way, that it does not cause any breaking changes for others (unless 
  agreed upon in an issue). 

## This PR solves the following problem**. 

resolve #178 

## How to test this PR**. 

For example:  
1. Open `main.tsx` files located in the `./development directory`.
2. Change the `locale` configuration in both `createCalendar` and `createDatePicker` to:
```
locale: 'ko-KR'
```
3. Run the development server with the command:
```
npm run dev
```

### CODE REVIEW

**Comment by tomosterlund** [comment]
@Dodo9128 Thanks for opening this PR! Will have a look at this after work today 😊

File: packages/translations/src/locales/ko-KR/date-picker.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { DatePickerTranslations } from '../../types/date-picker.translations'
+
+export const datePickerKoKR: DatePickerTranslations = {
+  Date: '일자',
+  'MM/DD/YYYY': '년/월/일',
+  'Next month': '다음 달',
+  'Previous month': '이전 달',
+  'Choose Date': '날짜 선택',
+  'Select View': '선택',
```
Comment (tomosterlund) [inline packages/translations/src/locales/ko-KR/date-picker.ts:9]: Is this short for something? Or did you just miss out on adding the word for "view"?

Bildschirmfoto 2024-01-26 um 18 14 57

Comment (Dodo9128) [reply packages/translations/src/locales/ko-KR/date-picker.ts:9]: I thought that would be enough, but I think "보기 선택" would be better for accurate delivery. I'll change it

**Review by tomosterlund — APPROVED** [review]
Thanks again for contributing to the project! I'll release this as 1.11.0 in a moment.

**Pull request merged**